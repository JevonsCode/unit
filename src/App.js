import React from 'react';

function App () {
    return (
        <div className="app" title="jest enzyme" data-test="app-conta">
            jest!
        </div>
    );
}

export default App;
