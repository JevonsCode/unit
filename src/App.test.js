import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from './App';
Enzyme.configure({ adapter: new Adapter() });

it('renders without crashing', () => {
    // const div = document.createElement('div');
    // const container = div.getElementsByClassName("App");
    // expect(container.length).toBe(2)
    const wrapper = mount(<App />);
    // const container = wrapper.find("[data-test='app-container']");
    // expect(wrapper.find('.app').length).toBe(1);
    // expect(wrapper.find(".app").prop("title")).toBe("jest enzyme");
    // expect(wrapper.find("[data-test='app-container']").prop("title")).toBe("jest enzyme");
    // expect(container).toExist();
    // expect(container).toHaveProp("title", "jest enzyme");
    expect(wrapper).toMatchSnapshot();
});
