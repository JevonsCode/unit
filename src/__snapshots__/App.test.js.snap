// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<App>
  <div
    className="app"
    data-test="app-container"
    title="jest enzyme"
  >
    jest
  </div>
</App>
`;
